.github/workflows/dockerhub-latest-chart.yml

```diff
@@ -3,6 +3,12 @@ on:
   push:
     branches:
       - master
+
+permissions:
+  contents: read     # Needed to fetch the repository contents.
+  packages: write    # Needed to push the chart to DockerHub.
```

Comment:
It's working for push to the ghcr.io, and i think we done't need it due to we push to the dockerhub.

Reply:
```diff
---  permissions:
---  contents: read     # Needed to fetch the repository contents.
---  packages: write    # Needed to push the chart to DockerHub.

+++ permissions: read-all
```

I think we can ack the permissions of read-all .